### ISSUE TITLE
Take down this repo?

### ISSUE THREAD

**Issue by ck37** [dlab-berkeley/collaboratool#116]
I'm wondering if we should hide this repo since it's old and we don't want people to stumble upon it? And/or create a new branch with a "this is old" message that links to BCE and is set as the default?

**Comment by aculich** [comment]
Agreed, we should deprecate this, but I don't have admin access... though @davclark or @BagOfMostlyWater  probably does.

**Comment by aculich** [comment]
Also, probably needs to archive/deprecate/delete these two old repos:

https://github.com/dlab-berkeley/collaboratool-base

https://github.com/dlab-berkeley/collaboratool-archive
